upgrade.adoc

```diff
@@ -0,0 +1,494 @@
+:product-name: Zimbra Collaboration
+:product-edition-commercial: Network Edition
+:product-short: Zimbra
+:product-family: Zimbra
+:product-provider: Zimbra
+:product-abbrev: ZCS
+:product-version: 8.8.0 GA
+:product-admin-guide: {product-name} Administration Guide
+:icons: font
+:title-logo-image: images/zimbra-logo-non-interlaced.png
+:showlinks:
+:source-highlighter: coderay
+:toc: left
+
+= {product-name} {product-version} Upgrade Steps
+
+== Before you upgrade
+
+The following tasks might need to be performed before you
+upgrade. After you review the tasks in this section, go to Upgrade
+Instructions. Be sure to read the
+https://wiki.zimbra.com/index.php?title=Zimbra_Releases/8.7.0[release note] information before upgrading.
+
+[NOTE]
+====
+https://bugzilla.zimbra.com/show_bug.cgi?id=105056[Bug
+105056]
+noted a problem that can occur during a rolling upgrade if two
+factor authentication (2FA) is enabled before all mailbox servers have
+been upgraded to 8.7.  In particular, pre-8.7 mailbox servers are not
+compatible with 2FA.
+
+Accordingly, it is recommended that 2FA is not enabled until all
+mailbox servers have been upgraded to 8.7.
+====
+
+=== Database Integrity Checking
+
+Some customers have had corrupted databases prior to upgrade, and the
+upgrade has in some of those cases exacerbated the problem. In order
+to detect any corrupted databases as early as possible, we have added
+an optional step to check the MariaDB database with
+zmdbintegrityreport prior to making any system changes. You are
+prompted to decide if you would like to run the zmdbintegrityreport.
+
+The `zmdbintegrityreport` can take minutes to an hour to run,
+depending on your system size and disk bandwidth.
+
+[NOTE]
+====
+The `zmdbintegrityreport` is run on a weekly basis from cron on all
+zimbra-store nodes. Large sites can opt to disable this by setting
+`zmlocalconfig -e zmdbintegrityreport_disabled=TRUE`. If
+you choose to disable this, it is recommended that the integrity
+reports be run by hand during the your normal maintenance windows and
+prior to running any {product-abbrev} upgrades.
+====
+
+=== Preparing your operating system
+
+Before you upgrade {product-abbrev}, {product-provider} recommends that the operating system is
+updated with the latest patches that have been tested with {product-abbrev}.
+
+==== Ubuntu OS
+
+* Ubuntu 16.04 LTS Server Edition (64-bit)
+* Ubuntu 14.04 LTS Server Edition (64-bit)
+* Ubuntu 12.04.4 LTS Server Edition running the saucy (3.11) or later
+  kernel is required. TODO - do we still support this?
++
+[CAUTION]
+====
+If the original install was done with Ubuntu 12.04.2 or earlier,
+manual intervention is required to switch to the saucy (3.11) or later
+kernel series. See https://wiki.ubuntu.com/Kernel/LTSEnablementStack
+for further information.
+====
++
+
+You can find your current kernel version by running:
```

Comment:
This and the next couple of bits look to have gotten disconnected from the list entry for Ubuntu 12.04 in the HTML rendering - and I think they should be part of that entry.